### PR TITLE
Add head tag

### PR DESCRIPTION
**Pull request by sophieheb**

### CODE REVIEW

File: src/components/Head.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  const router = useRouter();
+  return (
+    <NextHead>
+      <title>
+        {title}
+        {' '}
+        | YSM
+      </title>
+      <meta name="description" content={description} />
+      <meta property="og:url" content={`${process.env.BASE_URL} ${router.pathname}`} />
```
Comment (jits) [inline src/components/Head.js:18]: Is the space between `process.env.BASE_URL` and `router.pathname` expected/needed?

File: src/components/Head.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+      <meta property="og:type" content="website" />
+      <meta property="og:description" content={ogDescription || description} />
+      <meta property="og:title" content={ogTitle || title} />
+      <meta property="og:image" content={ogImage} />
```
Comment (jits) [inline src/components/Head.js:22]: What happens if `ogImage` is not set / is blank?

File: src/components/Head.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+      <meta property="og:site_name" content="YSM" />
+      <meta name="twitter:title" content={ogTitle || title} />
+      <meta name="twitter:description" content={ogDescription || description} />
+      <meta name="twitter:image" content={ogImage} />
+      <meta name="twitter:image:alt" content="Line drawing" />
```
Comment (jits) [inline src/components/Head.js:27]: The content value here should probably be dynamic too?

File: src/pages/_app.js [hunk 4]
```diff
@@ -100,10 +100,10 @@ function App({ Component, pageProps }) {
...
   return (
     <>
-      <Head>
-        <title>My page</title>
-        <meta name="viewport" content="minimum-scale=1, initial-scale=1, width=device-width" />
-      </Head>
+      <Head
+        title="Your Story Matters"
+        ogImage="logo.png"
+      />
```
Comment (jits) [inline src/pages/_app.js:106]: If `Head` is also declared in the app template (as well as individual pages), won't the tags be duplicated? (unless I'm missing something here).

File: src/pages/resources/[resourceSlug].js [hunk 5]
```diff
@@ -54,6 +55,7 @@ const ResourcePage = ({ propResource, propTheme, previewMode }) => {
       { theme && resource
         ? (
           <>
+            <Head title={resource.title} ogImage={resource.image ? resource.image.filename : '/logo.png'} />
```
Comment (jits) [inline src/pages/resources/[resourceSlug].js:58]: Maybe we could set the `logo.png` fallback in your `Head` component instead? And then allow null/blank image to be passed.

**Review by jits — APPROVED** [review]
Couple of suggestions.

File: src/components/Head.js (hunk 2 above)
Comment (sophieheb) [reply src/components/Head.js:22]: Have set logo.png as a backup, but if it was blank google/fb/twitter would choose an image from inside the page.

File: src/components/Head.js (hunk 3 above)
Comment (sophieheb) [reply src/components/Head.js:27]: Included

File: src/components/Head.js (hunk 1 above)
Comment (sophieheb) [reply src/components/Head.js:18]: Removed

File: src/pages/_app.js (hunk 4 above)
Comment (sophieheb) [reply src/pages/_app.js:106]: Next/Head overwrites them, so it gives us a default whilst we're waiting for content to load etc...

File: src/pages/resources/[resourceSlug].js (hunk 5 above)
Comment (sophieheb) [reply src/pages/resources/[resourceSlug].js:58]: Done!

**Pull request merged**